common_benchmark_suite/openxla/benchmark/comparative_suite/tf/model_definitions.py

```diff
@@ -62,9 +88,90 @@
+                source_url=string.Template(
+                    PARENT_GCS_DIR +
+                    "BERT_LARGE_FP32_TF_384XI32_BATCH${batch_size}/xla_hlo_before_optimizations.txt"
+                ),
+            ),
+        def_types.ModelArtifactType.TF_SAVEDMODEL_V2:
+            utils.ModelArtifactTemplate(
+                artifact_type=def_types.ModelArtifactType.TF_SAVEDMODEL_V2,
+                source_url=string.Template(
+                    PARENT_GCS_DIR +
+                    "BERT_LARGE_FP32_TF_384XI32_BATCH${batch_size}/tf-model.tar.gz"
+                ),
+            ),
     })
-
 BERT_LARGE_FP32_TF_384XI32_BATCHES = utils.build_batch_models(
     template=BERT_LARGE_FP32_TF_384XI32_BATCH_TEMPLATE,
     batch_sizes=[1, 16, 24, 32, 48, 64, 512, 1024, 1280])
+
+# ResNet50 models.
+# Model implementation from https://huggingface.co/docs/transformers/model_doc/resnet#transformers.TFResNetModel.
+# Batch sizes from MLPerf A100 Configs: https://github.com/mlcommons/inference_results_v2.1/tree/master/closed/NVIDIA/configs/resnet50
+RESNET_TF_IMPL = def_types.ModelImplementation(
+    id=unique_ids.MODEL_IMPL_RESNET_TF,
+    name="RESNET_TF",
+    tags=["cnn", "resnet"],
+    framework_type=def_types.ModelFrameworkType.TF_V2,
+    module_path=f"{utils.MODELS_MODULE_PATH}.tf.resnet.resnet_model",
+    source_info=
+    "https://huggingface.co/docs/transformers/model_doc/resnet#transformers.TFResNetModel",
+)
+RESNET50_FP32_TF_3X224X224XF32_BATCH_TEMPLATE = utils.ModelTemplate(
+    id=utils.BATCH_ID(unique_ids.MODEL_RESNET50_FP32_TF_3X224X224XF32),
+    name=utils.BATCH_NAME("RESNET50_FP32_TF_3X224X224XF32"),
+    tags=[utils.BATCH_TAG, "fp32"],
+    model_impl=RESNET_TF_IMPL,
+    model_parameters={
+        "batch_size": utils.BATCH_SIZE_PARAM,
+        "data_type": "fp32",
+        "model_name": "microsoft/resnet-50",
+    },
+    artifacts={
+        def_types.ModelArtifactType.STABLEHLO:
+            utils.ModelArtifactTemplate(
+                artifact_type=def_types.ModelArtifactType.STABLEHLO,
+                source_url=string.Template(
+                    PARENT_GCS_DIR +
+                    "RESNET50_FP32_TF_3X224X224XF32_BATCH${batch_size}/stablehlo.mlirbc"
+                ),
+            ),
+        def_types.ModelArtifactType.XLA_HLO_DUMP:
+            utils.ModelArtifactTemplate(
+                artifact_type=def_types.ModelArtifactType.XLA_HLO_DUMP,
+                source_url=string.Template(
+                    PARENT_GCS_DIR +
+                    "RESNET50_FP32_TF_3X224X224XF32_BATCH${batch_size}/xla_hlo_before_optimizations.txt"
+                ),
+            ),
+        def_types.ModelArtifactType.TF_SAVEDMODEL_V2:
+            utils.ModelArtifactTemplate(
+                artifact_type=def_types.ModelArtifactType.TF_SAVEDMODEL_V2,
+                source_url=string.Template(
+                    PARENT_GCS_DIR +
+                    "RESNET50_FP32_TF_3X224X224XF32_BATCH${batch_size}/tf-model.tar.gz"
```

Comment:
nit. We usually use `.tgz` instead of `.tar.gz`

Reply:
Replaced